code/mathematical-algorithms/Check_is_square/FindingISquare_binary_search.cpp

```diff
@@ -0,0 +1,51 @@
+#include <bits/stdc++.h>
```

Comment:
Don't include this. See #1915.

Reply:
remove and replaced by iostream header file